code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -33,6 +33,8 @@
 	mob_swap_flags = HUMAN|SIMPLE_ANIMAL|SLIME|MONKEY
 	mob_push_flags = ALLMOBS
 
+	meat_type = 0

```

Comment:
The code never checks if meat_type is an actual type of meat. I've tested it and it works fine.